.env.local

```
@@ -507,42 +507,62 @@
+# Simple JWT validation for development (using HS256 with shared secret)
+MCP_AUTH_JWT_AUDIENCE=http://localhost:3001/sse
+MCP_AUTH_JWT_ISSUER=gauzy-dev-auth
+MCP_AUTH_JWT_ALGORITHMS=HS256
+MCP_AUTH_JWT_PUBLIC_KEY=dev-secret-key-for-testing-only
+
+# Cache settings
+MCP_AUTH_TOKEN_CACHE_TTL=300
+MCP_AUTH_METADATA_CACHE_TTL=3600
+
+# Authorization server configuration (mock for development)
+MCP_AUTH_SERVERS='[{"issuer":"gauzy-dev-auth","authorizationEndpoint":"http://localhost:3001/oauth2/authorize","tokenEndpoint":"http://localhost:3001/oauth2/token","grantTypesSupported":["authorization_code","client_credentials","refresh_token"],"responseTypesSupported":["code"],"scopesSupported":["mcp.read","mcp.write","mcp.admin"],"codeChallengeMethodsSupported":["S256"]}]'
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)